ember/app/components/project-detailed/component.js

```diff
@@ -18,6 +18,10 @@ export default class ProjectDetailedComponent extends Component {
       );
       if (request.ok) {
         this.notification.success('Project synced successfully');
+        this.store.findRecord('project', this.args.project.id, {
+          include:
+            'versionedDependencies,versionedDependencies.releaseVersion,versionedDependencies.releaseVersion.dependency',
+        });
```

Comment:
Is the project in the response of the sync API call? If yes, you could just load that answer into the store instead of re-fetching it

Reply:
It does not return it
https://github.com/adfinis/Outdated/blob/085884569e3e67a3559cc49750e7c0c56e5ef384/api/outdated/outdated/views.py#L23-L32

Reply:
You use your usual serializer to return the updated project in order to use that data in the frontend.

Reply:
The `sync` endpoint now returns the updated project: https://github.com/adfinis/Outdated/pull/69/commits/445a3e364f324ea37afe8c3bbc4c52be939e6dab
Ember now loads the updated project from the response of the `sync` endpoint: https://github.com/adfinis/Outdated/pull/69/commits/0ba466a2963659b9ab7ab4cd8f0f29b1f2c686f5

Reply:
Nice work!